src/containers/BookList.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Book from '../components/Book';
+import { connect } from 'react-redux';
+import '../components/App.css';
+
+class BookList extends React.Component {
+	
+  render (){
+  	
+    return (<table>
+    	       <tbody>
+    	       <tr>
+                  <th>ID</th>
+                  <th>TITLE</th>
+                  <th>CATEGORY</th>
+                </tr>
+    	       {this.props.mybooks.map((elm, index) => {
+    	       	 return (<Book key = {index} value = {elm} />)
```

Comment:
Using array index as key is not recommended as it needs to uniquely identify elements. See [this article](https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318) on it.
`book.id` would be a great key here instead.